app/src/main/java/edu/grinnell/grinnell_publications_android/Activities/MainActivity.java

```diff
@@ -34,6 +37,7 @@
     @Bind(R.id.navigation_view) NavigationView navigationView;
```

Comment:
Nit-picking here again: How about we make all our member variables in this format : eg. `NavigationView nagivationView` will now become `NavigationView mNavigationView`. This way, we can easily differentiate b/n member variable and non-member variables.